### ISSUE TITLE
how to use $q 

### ISSUE THREAD

**Issue by fyeeme** [ekoeryanto/vite-quasar#1]
i import Notify and want use it ,but $q is undefined. how can i use $q 

to make clean, const $q = useQuasar(), will occer an Error, cause useQuasar is not export after auto import

**Comment by fyeeme** [comment]
I'v got the answer

**Comment by ekoeryanto** [comment]
hi, you can the useQuasar method.

**Comment by fyeeme** [comment]
yes, but useQuasar is not export after auto import ,so a pr is wait to be merged!.
